### DISCUSSION TITLE
Change highlight height and do not copy content when adding annotation

### DISCUSSION THREAD

**Discussion by Name-None** [sumatrapdfreader/sumatrapdf#3618] [Q&A]
Hi, I have two questions or suggestions about using Sumatra PDF.
Firstly, the current highlight height is too high for me, and looks a little uncomfortable. I hope you can add a parameter in the advanced options to adjust the highlight height. 
Secondly, when I select a paragraph of text and add a comment (such as highlight), the text will be copied to the clipboard (for Windows systems, the history of the clipboard can be seen). I don't want to copy this content every time because it makes it difficult for me to find what I need in the history of the clipboard.

**Comment by GitHubRulesOK** [comment]
The height is (generally following Adobe Standard and MuPDF implementation, hence a few quirks) usually set by the underlying font hence in some cases when the font is badly defined the height is almost non existent and the font needs correction.

Traditionally a highlight encompasses such text and OPTIONALLY may or may not copy the text into comment, at one time SumatraPDF did paste the clipped text into the comment field, but currently does not, so it is optional to use the selection as a comment, IF it was not placed in the clipboard it would then require lots of extra work for those that do want to use it, (manually or programmatically)
